src/SophiApp/Localizations/IT.xaml

```diff
@@ -72,21 +72,21 @@
     <system:String x:Key="Localization.Search">Cerca</system:String>
     <system:String x:Key="Localization.Toast.Title.Notificaton">Notifiche</system:String>
     <system:String x:Key="Localization.Toast.Update.Download">Download</system:String>
-    <system:String x:Key="Localization.Toast.Update.Header">un aggiornamento per la SophiApp è disponibile</system:String>
-    <system:String x:Key="Localization.View.NoData">Non ci sono dati da mostrare.</system:String>
-    <system:String x:Key="Localization.ViewChanges.Cancel">Cancella</system:String>
+    <system:String x:Key="Localization.Toast.Update.Header">È disponibile un aggiornamento di SophiApp</system:String>
+    <system:String x:Key="Localization.View.NoData">Non ci sono dati da visualizzare.</system:String>
+    <system:String x:Key="Localization.ViewChanges.Cancel">Annulla</system:String>
     <system:String x:Key="Localization.ViewChanges.Apply">Applica</system:String>
     <system:String x:Key="Localization.ViewChanges.Changed">Modificato</system:String>
     <system:String x:Key="Localization.ViewChanges.Setting">Impostazioni</system:String>
-    <system:String x:Key="Localization.ViewGames.NoData">Non ci sono dati da mostrare,</system:String>
+    <system:String x:Key="Localization.ViewGames.NoData">Non ci sono dati da visualizzare,</system:String>
     <system:String x:Key="Localization.ViewGames.PackagesRemoved">Oppure i pacchetti necessari sono stati rimossi.</system:String>
     <system:String x:Key="Localization.ViewSearch.NotFound">Non è stato trovato nulla</system:String>
     <system:String x:Key="Localization.ViewSearch.Running">Ricerca...</system:String>
     <system:String x:Key="Localization.ViewUwpApps.ForAllUsers">Per tutti gli utenti</system:String>
-    <system:String x:Key="Localization.ViewUwpApps.ForAllUsers.Description">Se disintalli una UWP app per tutti gli utenti, i nuovi account creati non includeranno la UWP app rimossa. Potrai ripristinarla solo sull'account corrente scaricando il pacchetto di intsallazione o scaricandola dal Microsoft Store, ma l'intsallazione avrà effetto solo sul'utente corrente.</system:String>
-    <system:String x:Key="Localization.Element.ContextMenu.Description.Copy">Copia descriptizione</system:String>
+    <system:String x:Key="Localization.ViewUwpApps.ForAllUsers.Description">Se disinstalli un'app  UWP per tutti gli utenti, i nuovi account creati non includeranno l'app UWP rimossa. Potrai ripristinarla solo nell'account attuale scaricando il pacchetto di installazione o scaricandola dal Microsoft Store, ma l'installazione avrà effetto solo per l'utente attuale.</system:String>
```

Comment:
Se disinstalli un app UWP per tutti gli utenti, i nuovi account utente creati non la includeranno. Potrai ripristinarla solo scaricando il pacchetto di installazione o tramite il Microsoft Store, ma l'installazione avrà effetto solo per l'utente in uso.

Reply:
Please take care that "app" in italian is female.  Then "un app" is not right.
It should use "una app" or "un'app". Sounds better the first.
OK for remove first "utente attuale"
Please consider to add a newline to separate paragraph for a bettere readibility of the long string.

Se disinstalli un'app UWP per tutti gli utenti, i nuovi account creati non includeranno l'app UWP rimossa.\nPotrai ripristinarla scaricando il pacchetto di installazione o scaricandola dal Microsoft Store, ma l'installazione avrà effetto solo per l'utente attuale.

Reply:
> Please consider to add if you can a newline (\n) to separate the two paragraphs for a better readibility of the long string.

@farag2 this doesn't give any problem, Right?

Reply:
OK. Thanks.

Reply:
> farag

As I remember, the current version doesn't accept this char. I will by the release of 2.0. I know about this.

Reply:
Not a big problem, should be pretty readable without the /n